packages/gatsby-source-drupal/README.md

```diff
@@ -156,6 +156,7 @@ module.exports = {
         baseUrl: `https://live-contentacms.pantheonsite.io/`,
         apiBase: `api`, // optional, defaults to `jsonapi`
         concurrentFileRequests: 60, // optional, defaults to `20`
+        secret: `xxxx-xxxx-xxxxx`, // optional, must match Drupal instance preview secret
```

Comment:
Should this exist in an environment variable rather than committed in code for security purposes?

Reply:
totally agree, Ill update the documentation to reflect that.

Reply:
Thanks, I think the documentation is much clearer and consistent with the rest of the readme file now.